uploadNeuroDB/NeuroDB/MRI.pm

```diff
@@ -1204,10 +1360,16 @@ sub make_jiv {
 }
 
 =pod
-Creates NIfTI files associated with MINC files and append
-its path to the parameter_file table using the parameter_type
-"check_nii_filename".
+
+=head3 make_nii($fileref, $data_dir)
+
+Creates NIfTI files associated with MINC files and append its path to the
+parameter_file table using the parameter_type "check_nii_filename".
+
+INPUTS: file hashref and data directory (typically C</data/project/data>)
```

Comment:
this should have been $PROJECT to be consistent, and 2 lines above, the tables should have been in C<>, but am merging anyway as this is becoming too picky (and no poison please!)

Reply:
@MounaSafiHarab At the bottom of the perldoc redmine ticket, I added the section below. We could then update it every time we encounter something not harmonized. You can add to it as I may have missed a few items. Couldn't remember anything else than what's below...

TODO once everything is merged to harmonize:
- remove empty todo sections and empty bugs sections
- harmonize /data/project/data/ to be /data/$PROJECT/data
- harmonize authorship for the LORIS team

Reply:
perfect!
I only remember item 3) in your list!
one thing we can add is that we need to ensure that all tables and files and variables are in C<> format...